.travis.yml

```diff
@@ -3,10 +3,18 @@ python:
   - 3.4
 addons:
   apt:
+    sources:
+      - ubuntu-toolchain-r-test
```

Comment:
Yeah, I see "ubuntu-toolchain-r" in the Ubuntu docs, but not "ubuntu-toolchain-r-test". It isn't obvious to me why you're using "-test". The docs do say that gcc 4.9 can be found in the test PPA, but you're installing 4.8 anyways. Maybe it's not important; I'm not sure. But that's why I'm asking :)